assets/app.js

```diff
@@ -165,6 +165,8 @@ new Vue({
         show_translator_components: false,
         show_translator_materials: false,
         show_translator_metatags: false,
+        baseProductStatus:true,
```

Comment:
Please clarify your intention of `baseProductStatus`.
Once its set to `false` are we setting `this.baseProductStatus = true` somewhere besides initially in data?